cassandra-scheduler/src/test/java/com/mesosphere/dcos/cassandra/scheduler/plan/upgradesstable/UpgradeSSTableStepTest.java

```diff
@@ -51,7 +51,7 @@ public void testInitial() {
         Mockito.when(cassandraState.get(UPGRADESSTABLE_NODE_0)).thenReturn(Optional.empty());
         final UpgradeSSTableContext context = UpgradeSSTableContext.create(Collections.emptyList(),
                 Collections.emptyList(), Collections.emptyList());
-        final UpgradeSSTableBlock block = UpgradeSSTableBlock.create(
+        final UpgradeSSTableStep block = UpgradeSSTableStep.create(
```

Comment:
Can you also replace s/block/step in this file and all the other test files too?